### PR TITLE
fix: API call to TF Registry always requires the namespace of the TF provider

### PR DESCRIPTION
**Pull request by ansgarm**

### CODE REVIEW

File: src/scripts/check-for-upgrades.ts [hunk 1]
```diff
@@ -22,10 +22,15 @@ export class CheckForUpgradesScriptFile extends FileBase {
   }
 
   protected synthesizeContent(resolver: IResolver): string | undefined {
-    const { providerVersion, fqproviderName } = resolver.resolve(
+    let { providerVersion, fqproviderName } = resolver.resolve(
       this.options
     ) as CheckForUpgradesScriptFileOptions;
 
+    // aws -> hashicorp/aws (else the registry API call fails)
+    if (fqproviderName.indexOf("/")) {
```
Comment (mutahhir) [inline src/scripts/check-for-upgrades.ts:30]: How does this work? From my understanding, the result of indexOf would either be > 0 or < 0, and both would resolve to truthy. Did you mean?

```suggestion
    if (fqproviderName.indexOf("/") < 0) {
```

Comment (ansgarm) [reply src/scripts/check-for-upgrades.ts:30]: Good catch 😅

Comment (ansgarm) [reply src/scripts/check-for-upgrades.ts:30]: I think I was intending to use this:
```suggestion
    if (fqproviderName.includes("/")) {
```

File: src/scripts/check-for-upgrades.ts [hunk 2]
```diff
@@ -22,10 +22,15 @@ export class CheckForUpgradesScriptFile extends FileBase {
   }
 
   protected synthesizeContent(resolver: IResolver): string | undefined {
-    const { providerVersion, fqproviderName } = resolver.resolve(
+    let { providerVersion, fqproviderName } = resolver.resolve(
       this.options
     ) as CheckForUpgradesScriptFileOptions;
 
+    // aws -> hashicorp/aws (else the registry API call fails)
+    if (fqproviderName.includes("/")) {
```
Comment (ansgarm) [inline src/scripts/check-for-upgrades.ts:30]: whoops
```suggestion
    if (!fqproviderName.includes("/")) {
```

**Review by mutahhir — APPROVED** [review]
Nice, 🚢

**Pull request merged**